src/components/MovieCard/MovieCard.tsx

```diff
@@ -12,7 +12,7 @@ export const MovieCard: React.FC<Props> = ({ movie }) => (
       <figure className="image is-4by3">
         <img
           data-cy="moviePoster"
-          src={movie.imgUrl}
+          src={movie?.imgUrl}
```

Comment:
```suggestion
          src={imgUrl}
```
do destructuring assignment above